deploy/manifests/deployment.yaml

```diff
@@ -208,6 +208,7 @@ spec:
             - --timeout=2m5s
             - --leader-election
             - --leader-election-namespace=$(POD_NAMESPACE)
+            - --default-fstype=ext4
```

Comment:
The argument is appended to the attacher pod. It should be used by the csi-provisioner